ckan/common.py

```diff
@@ -147,13 +147,61 @@ def _get_request():
     return flask.request
 
 
+class HtmxDetails(object):
```

Comment:
```suggestion
class HTMXDetails(object):
```
would follow the python naming conventions better, e.g. https://docs.python.org/3/library/html.parser.html?highlight=html#html.parser.HTMLParser

Reply:
unless htmx isn't initials for something... :thinking:

Reply:
`htmx` are not initials and it is always referred as `htmx` (lower case) in comparison with `HTML`.

Their docs always refer as `htmx`. Tha's why I used `HtmxDetails`. That's how `django-htmx` calls it so I'm following them as well: https://django-htmx.readthedocs.io/en/latest/middleware.html